kolibri/auth/filters.py

```diff
@@ -99,6 +92,10 @@ def _as_sql_reference(self, ref):
         else:
             raise Exception("Not a valid reference: %r" % ref)
```

Comment:
I guess I missed this in #50, but it is an anti-pattern IMO to raise plain Exceptions. If the exception in this case defies categorization, I would say it's still better to raise a `kolibri.core.errors.KolibriError`.